gtsaruk/02_HTML/task2.2.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>  
+<HTML> 
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Календарь</TITLE>
+	</HEAD> 
+	<BODY>
+		<TABLE width="300" border="3" align="center">
+			<CAPTION>Месяц моего рождения</CAPTION>
+			<col width="25%"><col width="15%"><col width="15%"><col width="15%"><col width="15%"><col width="15%">
+		<TR>
+			<TH colspan="6">Ноябрь</TH>
+		</TR>
+			<TD align="center"><b>ПН</b></TD>
```

Comment:
here should be th tag, could you please check all cells with week days and month here